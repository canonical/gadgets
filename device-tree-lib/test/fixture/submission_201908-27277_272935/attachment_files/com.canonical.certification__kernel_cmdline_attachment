 coherent_pool=1M snd_bcm2835.enable_compat_alsa=0 snd_bcm2835.enable_hdmi=1 video=Composite-1:720x480@60i,margin_left=32,margin_right=32,margin_top=32,margin_bottom=32 smsc95xx.macaddr=B8:27:EB:E8:E8:15 vc_mem.mem_base=0x3ec00000 vc_mem.mem_size=0x40000000  dwc_otg.lpm_enable=0 console=ttyAMA0,115200 elevator=deadline rng_core.default_quality=700 vt.handoff=2 quiet splash quiet splash snapd_recovery_mode=run snapd_recovery_system=20220716 panic=-1
