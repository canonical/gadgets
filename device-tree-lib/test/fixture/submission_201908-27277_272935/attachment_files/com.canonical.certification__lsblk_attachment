KNAME="loop0" TYPE="loop" MOUNTPOINT="/snap/core20/1584"
KNAME="loop1" TYPE="loop" MOUNTPOINT="/snap/pi-kernel/497"
KNAME="loop2" TYPE="loop" MOUNTPOINT="/run/mnt/snapd"
KNAME="loop3" TYPE="loop" MOUNTPOINT="/snap/snapd/16549"
KNAME="loop4" TYPE="loop" MOUNTPOINT="/snap/pi/106"
KNAME="loop5" TYPE="loop" MOUNTPOINT="/snap/checkbox/1849/checkbox-runtime"
KNAME="loop6" TYPE="loop" MOUNTPOINT="/snap/checkbox/1849"
KNAME="mmcblk0" TYPE="disk" MOUNTPOINT=""
KNAME="mmcblk0p1" TYPE="part" MOUNTPOINT="/run/mnt/ubuntu-seed"
KNAME="mmcblk0p2" TYPE="part" MOUNTPOINT="/run/mnt/ubuntu-boot"
KNAME="mmcblk0p3" TYPE="part" MOUNTPOINT="/run/mnt/ubuntu-save"
KNAME="mmcblk0p4" TYPE="part" MOUNTPOINT="/run/mnt/base/writable"
KNAME="mmcblk0boot0" TYPE="disk" MOUNTPOINT=""
KNAME="mmcblk0boot1" TYPE="disk" MOUNTPOINT=""
