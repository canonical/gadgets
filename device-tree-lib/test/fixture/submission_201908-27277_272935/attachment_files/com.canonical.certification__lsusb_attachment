Bus 001 Device 001 ID 1d6b:0002 Linux 5.4.0-1066-raspi dwc_otg_hcd DWC OTG Controller 3f980000.usb
Bus 001 Device 002 ID 0bda:8153 Realtek USB 10/100/1000 LAN 000001
