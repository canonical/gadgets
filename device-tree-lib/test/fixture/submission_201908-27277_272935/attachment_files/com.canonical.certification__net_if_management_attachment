==enx00e04c68e179==
has netplan
