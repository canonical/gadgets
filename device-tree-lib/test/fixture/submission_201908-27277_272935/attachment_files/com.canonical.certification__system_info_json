{
    "System": [
        "Kernel: 5.4.0-1066-raspi armv7l",
        "Distro: Ubuntu Core 20"
    ],
    "Machine": [
        "Type: ARM Device System: Raspberry Pi Compute Module 3 Plus Rev 1.0 details: BCM2835 rev: a02100 "
    ],
    "Memory": [
        "920 MiB"
    ],
    "Audio": [
        "bcm2835-audio",
        "bcm2835-hdmi"
    ],
    "Graphics": [
        "bcm2835-vc4",
        "bcm2835-hdmi"
    ],
    "Network": [
        "bcm2835-sdhci",
        "Realtek USB 10/100/1000 LAN"
    ],
    "Drives": [
        "8GTF4R 7.28 GiB "
    ]
}
