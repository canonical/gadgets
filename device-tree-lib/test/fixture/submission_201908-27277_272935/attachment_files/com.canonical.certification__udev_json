[
    {
        "bus": "video4linux",
        "driver": "bcm2835-codec",
        "name": "video10",
        "path": "/devices/platform/soc/3f00b840.mailbox/bcm2835-codec/video4linux/video10",
        "product": "bcm2835-codec-decode",
        "product_slug": "bcm2835-codec-decode"
    },
    {
        "bus": "video4linux",
        "driver": "bcm2835-codec",
        "name": "video11",
        "path": "/devices/platform/soc/3f00b840.mailbox/bcm2835-codec/video4linux/video11",
        "product": "bcm2835-codec-encode",
        "product_slug": "bcm2835-codec-encode"
    },
    {
        "bus": "video4linux",
        "driver": "bcm2835-codec",
        "name": "video12",
        "path": "/devices/platform/soc/3f00b840.mailbox/bcm2835-codec/video4linux/video12",
        "product": "bcm2835-codec-isp",
        "product_slug": "bcm2835-codec-isp"
    },
    {
        "bus": "video4linux",
        "driver": "bcm2835-isp",
        "name": "video13",
        "path": "/devices/platform/soc/3f00b840.mailbox/bcm2835-isp/video4linux/video13",
        "product": "bcm2835-isp",
        "product_slug": "bcm2835-isp"
    },
    {
        "bus": "video4linux",
        "driver": "bcm2835-isp",
        "name": "video14",
        "path": "/devices/platform/soc/3f00b840.mailbox/bcm2835-isp/video4linux/video14",
        "product": "bcm2835-isp",
        "product_slug": "bcm2835-isp"
    },
    {
        "bus": "video4linux",
        "driver": "bcm2835-isp",
        "name": "video15",
        "path": "/devices/platform/soc/3f00b840.mailbox/bcm2835-isp/video4linux/video15",
        "product": "bcm2835-isp",
        "product_slug": "bcm2835-isp"
    },
    {
        "bus": "video4linux",
        "driver": "bcm2835-isp",
        "name": "video16",
        "path": "/devices/platform/soc/3f00b840.mailbox/bcm2835-isp/video4linux/video16",
        "product": "bcm2835-isp",
        "product_slug": "bcm2835-isp"
    },
    {
        "bus": "vchiq",
        "category": "MMAL",
        "driver": "bcm2835_vchiq",
        "name": "vchiq",
        "path": "/devices/platform/soc/3f00b840.mailbox/vchiq/vchiq"
    },
    {
        "bus": "watchdog",
        "category": "WATCHDOG",
        "driver": "bcm2835-wdt",
        "name": "watchdog0",
        "path": "/devices/platform/soc/3f100000.watchdog/bcm2835-wdt/watchdog/watchdog0"
    },
    {
        "bus": "mmc",
        "category": "DISK",
        "driver": "mmcblk",
        "name": "mmcblk0",
        "path": "/devices/platform/soc/3f202000.mmc/mmc_host/mmc0/mmc0:0001",
        "product": "8GTF4R",
        "product_slug": "8GTF4R"
    },
    {
        "bus": "mmc_rpmb",
        "category": "CARDREADER",
        "driver": "mmcblk",
        "name": "mmcblk0rpmb",
        "path": "/devices/platform/soc/3f202000.mmc/mmc_host/mmc0/mmc0:0001/mmcblk0rpmb",
        "product": "8GTF4R",
        "product_slug": "8GTF4R"
    },
    {
        "bus": "usb",
        "driver": "usb",
        "name": "bus/usb/001/001",
        "path": "/devices/platform/soc/3f980000.usb/usb1",
        "product": "2.0 root hub",
        "product_slug": "2.0_root_hub",
        "vendor": "Linux Foundation",
        "vendor_slug": "Linux_Foundation"
    },
    {
        "bus": "usb",
        "category": "OTHER",
        "driver": "hub",
        "path": "/devices/platform/soc/3f980000.usb/usb1/1-0:1.0",
        "product": "2.0 root hub",
        "product_id": 2,
        "product_slug": "2.0_root_hub",
        "vendor": "Linux Foundation",
        "vendor_id": 7531,
        "vendor_slug": "Linux_Foundation"
    },
    {
        "bus": "usb",
        "driver": "usb",
        "name": "bus/usb/001/002",
        "path": "/devices/platform/soc/3f980000.usb/usb1/1-1",
        "product": "RTL8153 Gigabit Ethernet Adapter",
        "product_slug": "RTL8153_Gigabit_Ethernet_Adapter",
        "vendor": "Realtek Semiconductor Corp.",
        "vendor_slug": "Realtek_Semiconductor_Corp."
    },
    {
        "bus": "usb",
        "category": "NETWORK",
        "driver": "r8152",
        "interface": "enx00e04c68e179",
        "mac": "00:e0:4c:68:e1:79",
        "path": "/devices/platform/soc/3f980000.usb/usb1/1-1/1-1:1.0",
        "product": "RTL8153 Gigabit Ethernet Adapter",
        "product_id": 33107,
        "product_slug": "RTL8153_Gigabit_Ethernet_Adapter",
        "vendor": "Realtek Semiconductor Corp.",
        "vendor_id": 3034,
        "vendor_slug": "Realtek_Semiconductor_Corp."
    },
    {
        "bus": "drm",
        "category": "VIDEO",
        "driver": "vc4-drm",
        "name": "dri/card0",
        "path": "/devices/platform/soc/soc:gpu/drm/card0",
        "product": "drm-platform-soc_gpu",
        "product_slug": "drm-platform-soc_gpu"
    }
]
