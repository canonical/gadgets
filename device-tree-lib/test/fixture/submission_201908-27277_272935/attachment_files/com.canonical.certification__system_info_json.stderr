No devices found
No devices found
No devices found
No devices found




